starter-code/css/main.css

```diff
@@ -6,3 +6,171 @@ Green: #00B172
 White: #FFF
 
 */
+
+body {
+  margin: 0;
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
+  font-weight: 300;
+}
+
+/* HEADER */
+
+header {
+  margin: 20px auto;
```

Comment:
by adding 

```
header {
position: fixed;
}
```

your header would remain in place when you scroll. You'd also need to change the with to 100%. 

it's also better to increase the height of the header rather than adding a margin. Adding margin can be useful sometimes, but can also interfere with other css styling later. So here you could just set the height to 60px